lte/gateway/python/integ_tests/s1aptests/test_eps_bearer_context_status_ded_bearer_deact.py

```diff
@@ -0,0 +1,271 @@
+            )
+            act_ded_ber_ctxt_req = response.cast(
+                s1ap_types.UeActDedBearCtxtReq_t,
+            )
+            self._s1ap_wrapper.sendActDedicatedBearerAccept(
+                ue_id, act_ded_ber_ctxt_req.bearerId,
+            )
+
+            # Create 2nd dedicated bearer
+            print(
+                "********************** Adding 2nd dedicated bearer to IMSI",
+                "".join([str(i) for i in req.imsi]),
+            )
+
+            flow_list.append(self._spgw_util.create_default_ipv4_flows())
+            self._spgw_util.create_bearer(
+                "IMSI" + "".join([str(i) for i in req.imsi]),
+                act_def_bearer_req.m.pdnInfo.epsBearerId,
+                flow_list[i],
+            )
+
+            response = self._s1ap_wrapper.s1_util.get_response()
+            self.assertEqual(
+                response.msg_type, s1ap_types.tfwCmd.UE_ACT_DED_BER_REQ.value,
+            )
+            act_ded_ber_ctxt_req = response.cast(
+                s1ap_types.UeActDedBearCtxtReq_t,
+            )
+            self._s1ap_wrapper.sendActDedicatedBearerAccept(
+                ue_id, act_ded_ber_ctxt_req.bearerId,
+            )
+
+        print("Sleeping for 5 seconds")
+        time.sleep(5)
+        # Verify if flow rules are created
+        for i in range(num_pdns):
+            dl_flow_rules = {
+                default_ip: [],
+                sec_ip[i]: [flow_list[i]],
+            }
+            # 1 UL flow is created per bearer
+            num_ul_flows = 7
+            self._s1ap_wrapper.s1_util.verify_flow_rules(
+                num_ul_flows, dl_flow_rules,
+            )
+
+        print(
+            "************************* Sending UE context release request ",
+            "for UE id ",
+            ue_id,
+        )
+        # Send UE context release request to move UE to idle mode
+        cntxt_rel_req = s1ap_types.ueCntxtRelReq_t()
+        cntxt_rel_req.ue_Id = ue_id
+        cntxt_rel_req.cause.causeVal = (
+            gpp_types.CauseRadioNetwork.USER_INACTIVITY.value
+        )
+        self._s1ap_wrapper.s1_util.issue_cmd(
+            s1ap_types.tfwCmd.UE_CNTXT_REL_REQUEST, cntxt_rel_req,
+        )
+        response = self._s1ap_wrapper.s1_util.get_response()
+        self.assertEqual(
+            response.msg_type, s1ap_types.tfwCmd.UE_CTX_REL_IND.value,
+        )
+        print(" Sleeping for 2 seconds")
+        time.sleep(2)
+        print(
+            "************************* Sending Tracking Area Update ",
+            "request for UE id ",
+            ue_id,
+        )
+        tau_req = s1ap_types.ueTauReq_t()
+        tau_req.ue_Id = ue_id
+        tau_req.type = s1ap_types.Eps_Updt_Type.TFW_TA_UPDATING.value
+        tau_req.Actv_flag = True
+        # Set default bearers 5,6 and 9 as active and
+        # dedicated bearers 7,8,10 and 11 as inactive
+        # epsBearerCtxSts IE is 16 bits
```

Comment:
For readability, could you please add a reference to the specification and the bitmap for the EPS Bearer Context Status value used in line 218?